T5_1/SparseMatrix.cs

```diff
@@ -0,0 +1,99 @@
+﻿using System.Collections;
+using System.Text;
+namespace Homework_5_1
+{
+	public class SparseMatrix : IEnumerable<long>
+	{
+		private Dictionary<(int, int), long> matrix;
```

Comment:
Implement public Get for _rows an _columns, change names according to code style for private fields